tests/functional/a/arguments_differ.py

```diff
@@ -3,13 +3,13 @@
 
 class Parent(object):
 
-    def test(self):

```

Comment:
Could we refrain from changing the existing functional test in this MR, please ? I would like to see if the old use cases are all handled well. We can add more functional tests if required :)